tests/fixtures/ecma-version/3/strict/use-strict.result.js

```diff
@@ -0,0 +1,259 @@
+export default {
+    "type": "Program",
+    "loc": {
+        "start": {
+            "line": 6,
+            "column": 0
+        },
+        "end": {
+            "line": 7,
+            "column": 13
+        }
+    },
+    "range": [
+        183,
+        210
+    ],
+    "body": [
+        {
+            "type": "ExpressionStatement",
+            "loc": {
+                "start": {
+                    "line": 6,
+                    "column": 0
+                },
+                "end": {
+                    "line": 6,
+                    "column": 13
+                }
+            },
+            "range": [
+                183,
+                196
+            ],
+            "expression": {
+                "type": "Literal",
+                "loc": {
+                    "start": {
+                        "line": 6,
+                        "column": 0
+                    },
+                    "end": {
+                        "line": 6,
+                        "column": 12
+                    }
+                },
+                "range": [
+                    183,
+                    195
+                ],
+                "value": "use strict",
+                "raw": "\"use strict\""
+            },
+            "directive": "use strict"
```

Comment:
I'm not sure if this should be reported to Acorn as a possible bug. As far as I know, ES3 didn't have the concept of directive prologues, so maybe no nodes should be parsed as [Directive](https://github.com/estree/estree/blob/master/es5.md#directive) nodes in ES3 mode.

Reply:
Agree. This seems like a bug in Acorn.